fdbclient/NativeAPI.actor.cpp

```diff
@@ -948,8 +948,13 @@ DatabaseContext::DatabaseContext(Reference<AsyncVar<Reference<ClusterConnectionF
 		            .withPrefix(SpecialKeySpace::getModuleRange(SpecialKeySpace::MODULE::CONFIGURATION).begin)));
 		registerSpecialKeySpaceModule(
 			SpecialKeySpace::MODULE::MANAGEMENT, SpecialKeySpace::IMPLTYPE::READWRITE,
-			std::make_unique<LockDatabaseImpl>(singleKeyRange(LiteralStringRef("dbLocked"))
+			std::make_unique<LockDatabaseImpl>(singleKeyRange(LiteralStringRef("db_locked"))
```

Comment:
If we want to commit to a snake case convention for special keys, maybe we can validate that? Maybe validate that there aren't any capital letters or something?

Reply:
Make sense.
Let me add the validation code in the registration process.

Reply:
Added the code to only allow small letter, numbers, `/`, `_` in the path after removing `\xff\xff` prefix and potential suffix(\xff\xff or \x00)